docs/installation.rst

```diff
@@ -1,32 +1,71 @@
-============
+.. _installation:
+
 Installation
 ============
 
 This package requires NumPy, which you can install alone or as a part of SciPy.
-`Some Python distributions <http://www.scipy.org/install.html#scientific-python-distributions>`_ come with NumPy/SciPy pre-installed. For Python distributions
-without NumPy/SciPy, Windows/Mac users should install
-`pre-compiled binaries of NumPy/SciPy <http://www.scipy.org/scipylib/download.html#official-source-and-binary-releases>`_, and Linux users may have NumPy/SciPy
-available in `their repositories <http://www.scipy.org/scipylib/download.html#third-party-vendor-package-managers>`_.
+`Some Python distributions <https://www.scipy.org/install.html>`_
+come with NumPy/SciPy pre-installed. For Python distributions without
+NumPy/SciPy, Windows/Mac users should install
+`pre-compiled binaries of NumPy/SciPy <https://www.scipy.org/scipylib/download.html#official-source-and-binary-releases>`_, and Linux users may have
+NumPy/SciPy available in
+`their repositories <https://www.scipy.org/scipylib/download.html#third-party-vendor-package-managers>`_.
+
+When you have NumPy, you may use either PyPi or GitHub to install this package.
+
 
-When you have NumPy, install this package at the command line using
+.. _installation-pip:
+
+PyPi
+----
+This is the most straigforward option!  From the command line use
 ``pip`` [1]_::
 
     pip install apexpy
 
-This assumes that the same version of libgfortran is installed in the same location as when the pip wheel was built. If not, you may have trouble importing apexpy and you will have to build apexpy yourself using:
+This assumes that the same version of libgfortran is installed in the same
+location as when the pip wheel was built (in the event that a wheel is
+available). If not, you may have trouble importing apexpy.  In the event that
+you run into issues, you can get around this problem by using
```

Comment:
```suggestion
In the event that you run into issues, you can get around this problem by using
```

I was planning on not uploading wheels this time, since they just ended up causing unnecessary problems.  I suppose that means the pip example just below is not needed.